[0.155s] Invoking command in '/home/lucasb/RBE500_Project/build/custom_messages': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/home/lucasb/RBE500_Project/install/Manipulator_X:/opt/ros/humble /usr/bin/cmake /home/lucasb/RBE500_Project/src/custom_messages -DCMAKE_INSTALL_PREFIX=/home/lucasb/RBE500_Project/install/custom_messages
[0.202s] -- The C compiler identification is GNU 11.4.0
[0.248s] -- The CXX compiler identification is GNU 11.4.0
[0.253s] -- Detecting C compiler ABI info
[0.310s] -- Detecting C compiler ABI info - done
[0.318s] -- Check for working C compiler: /usr/bin/cc - skipped
[0.318s] -- Detecting C compile features
[0.318s] -- Detecting C compile features - done
[0.321s] -- Detecting CXX compiler ABI info
[0.384s] -- Detecting CXX compiler ABI info - done
[0.391s] -- Check for working CXX compiler: /usr/bin/c++ - skipped
[0.391s] -- Detecting CXX compile features
[0.391s] -- Detecting CXX compile features - done
[0.392s] -- Found ament_cmake: 1.3.10 (/opt/ros/humble/share/ament_cmake/cmake)
[0.516s] -- Found Python3: /usr/bin/python3 (found version "3.10.12") found components: Interpreter 
[0.573s] -- Found geometry_msgs: 4.2.4 (/opt/ros/humble/share/geometry_msgs/cmake)
[0.610s] -- Found rosidl_generator_c: 3.1.5 (/opt/ros/humble/share/rosidl_generator_c/cmake)
[0.614s] -- Found rosidl_adapter: 3.1.5 (/opt/ros/humble/share/rosidl_adapter/cmake)
[0.620s] -- Found rosidl_generator_cpp: 3.1.5 (/opt/ros/humble/share/rosidl_generator_cpp/cmake)
[0.629s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.641s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.668s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[0.804s] CMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[0.804s]   execute_process(/usr/bin/python3 -m rosidl_adapter --package-name
[0.804s]   custom_messages --arguments-file
[0.804s]   /home/lucasb/RBE500_Project/build/custom_messages/rosidl_adapter__arguments__custom_messages.json
[0.804s]   --output-dir
[0.804s]   /home/lucasb/RBE500_Project/build/custom_messages/rosidl_adapter/custom_messages
[0.804s]   --output-file
[0.804s]   /home/lucasb/RBE500_Project/build/custom_messages/rosidl_adapter/custom_messages.idls)
[0.804s]   returned error code 1:
[0.804s] 
[0.804s]   Error processing 'geometry_msgs/Pose.msg pose' of
[0.804s]   'custom_messages/inv_kin_Request': ''Pose.msg' is an invalid message name.
[0.804s]   It should have the pattern '^[A-Z][A-Za-z0-9]*$''
[0.804s] 
[0.804s]   Traceback (most recent call last):
[0.804s] 
[0.804s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[0.805s]       return _run_code(code, main_globals, None,
[0.805s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[0.805s]       exec(code, run_globals)
[0.805s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.805s]       sys.exit(main())
[0.805s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[0.805s]       abs_idl_file = convert_to_idl(
[0.805s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 24, in convert_to_idl
[0.805s]       return convert_srv_to_idl(
[0.805s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/srv/__init__.py", line 28, in convert_srv_to_idl
[0.805s]       srv = parse_service_string(package_name, input_file.stem, content)
[0.805s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 859, in parse_service_string
[0.805s]       request_message = parse_message_string(
[0.805s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 520, in parse_message_string
[0.805s]       Type(type_string, context_package_name=pkg_name),
[0.805s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 277, in __init__
[0.805s]       super(Type, self).__init__(
[0.805s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 205, in __init__
[0.805s]       raise InvalidResourceName(
[0.805s] 
[0.805s]   rosidl_adapter.parser.InvalidResourceName: 'Pose.msg' is an invalid message
[0.805s]   name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$'
[0.805s] 
[0.805s] Call Stack (most recent call first):
[0.805s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[0.805s]   CMakeLists.txt:16 (rosidl_generate_interfaces)
[0.805s] 
[0.805s] 
[0.805s] -- Configuring incomplete, errors occurred!
[0.805s] See also "/home/lucasb/RBE500_Project/build/custom_messages/CMakeFiles/CMakeOutput.log".
[0.808s] Invoked command in '/home/lucasb/RBE500_Project/build/custom_messages' returned '1': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/home/lucasb/RBE500_Project/install/Manipulator_X:/opt/ros/humble /usr/bin/cmake /home/lucasb/RBE500_Project/src/custom_messages -DCMAKE_INSTALL_PREFIX=/home/lucasb/RBE500_Project/install/custom_messages
